.eslintrc.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-    "extends": "standard"
-};
\ No newline at end of file
+  "extends": "standard",
+  "parser": "babel-eslint",
```

Comment:
Switching to the babel-eslint parser means eslint can handle the async generators correctly now.

Reply:
Amazing ! I didn't know that!